bugwarrior/command.py

```diff
@@ -67,6 +74,8 @@ def pull(dry_run, flavor, interactive, debug):
         )
     except RuntimeError as e:
         log.critical("Aborted (%s)" % e)
+    except SystemExit:
+        pass
```

Comment:
I figured that was the rationale. Is there a way we can suppress the traceback while still throwing a bad exit code?